packages/ui/Scrollable/Scrollbar.js

```diff
@@ -197,53 +176,53 @@ class ScrollbarBase extends PureComponent {
  * @ui
  * @private
  */
-class Scrollbar extends Component {
-	static propTypes = /** @lends ui/Scrollable.Scrollbar.prototype */ {
-		/**
-		 * If `true`, the scrollbar will be oriented vertically.
-		 *
-		 * @type {Boolean}
-		 * @default true
-		 * @public
-		 */
-		vertical: PropTypes.bool
-	}
-
-	static defaultProps = {
-		vertical: true
-	}
-
-	setApi = (ref) => {
-		if (ref) {
-			const {getContainerRef, showThumb, startHidingThumb, update: uiUpdate} = ref;
-
-			this.getContainerRef = getContainerRef;
-			this.showThumb = showThumb;
-			this.startHidingThumb = startHidingThumb;
-			this.update = uiUpdate;
+const Scrollbar = forwardRef((props, ref) => {
+	const scrollbarBaseRef = useRef(null);
+
+	useImperativeHandle(ref, () => {
+		if (scrollbarBaseRef.current) {
```

Comment:
If `scrollbarBaseRef.current` is equal to `null`, then what about returning {} object? Because the `current` of the scrollbar ref could be accessed.

Reply:
Maybe we don't need to test its validity at all. I will remove _if_ statement for now. If we need to put it back, then we can consider returning an empty object.